mapper/mapper.go

```diff
@@ -58,6 +60,40 @@ func MainFigure(ctx context.Context, id, datePeriod string, figure zebedee.Times
 	return &mf
 }
 
+func ReleaseCalendar(rawReleaseCalendar release_calendar.ReleaseCalendar) *model.ReleaseCalendar {
+	rc := model.ReleaseCalendar{
+		Releases:         getLatestReleases(*rawReleaseCalendar.Result.Results),
+		NumberOfReleases: strconv.Itoa(rawReleaseCalendar.Result.NumberOfResults),
+	}
+	return &rc
+}
+
+func getLatestReleases(rawReleases []release_calendar.Results) []model.Release {
+	var latestReleases []model.Release
+
+	// Removed canceled releases
+	for i := len(rawReleases) - 1; i >= 0; i-- {
+		if rawReleases[i].Description.Cancelled {
+			rawReleases = append(rawReleases[:i], rawReleases[i+1:]...)
```

Comment:
Create new 'rawReleases' with everything up until the removed element, skip over it then everything else after.
Variadic used to prevent slice as single element being appended, instead, each element is being appended to the original slice as a new element